bot/cogs/memes_cog/memes_cog.py

```diff
@@ -95,5 +103,51 @@ async def spongebob(self, ctx, *, args):
 
         await ctx.send(embed=embed)
 
+    @commands.command(aliases=['rave', '🦀'])
+    async def crab(self, ctx, *, args='Bottom text\n is dead'):
+        # crab.gif dimensions - 352 by 200
```

Comment:
We need a py doc comment here to have something to show in the !help command when people run that. just a short description of the what and how